backend/tests/modules/auth/test_password_reset.py

```
@@ -0,0 +1,292 @@
+
+# --------------------------------------------------------------------------
+# /reset-password
+# --------------------------------------------------------------------------
+
+
+@pytest.mark.asyncio
+async def test_reset_password_happy_path_updates_password_and_marks_token_used(
+    async_client: AsyncClient, stub_send_email, db_session: Session
+):
+    email = f"reset_{uuid.uuid4().hex[:8]}@example.com"
+    old_password = "OldPassword123!"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "old_password"

(S105)